### DISCUSSION TITLE
Server can run more than one replica?

### DISCUSSION THREAD

**Discussion by eliasscosta** [woodpecker-ci/woodpecker#3133] [Ask a Question] [answered]
Hello, We can run a server instance with more than one replica? 

I'm trying to achieve some high availability with the server instance. But when I run 3 server instances and send a batch of webhooks just a few start the pipeline others remain with "not started yet".

Do we have some documentation related to it?

**Comment by zc-devs** [comment, marked as answer]
Perhaps, [this](https://woodpecker-ci.org/docs/administration/agent-config#workflows-per-agent) one as well as [this](https://woodpecker-ci.org/docs/administration/agent-config#woodpecker_server)

More details there: [Server doesn't work (nicely) with multiple replicas](https://github.com/woodpecker-ci/helm/issues/42)
